library/std/src/sys/sgx/ext/io.rs

```diff
@@ -71,18 +74,38 @@ impl AsRawFd for net::TcpListener {
     }
 }
 
+/// Metadata for `TcpStream`.
+#[derive(Debug, Clone, Default)]
+pub struct TcpStreamMetadata {
+    /// Local address of the TCP stream
+    pub local_addr: Option<String>,
```

Comment:
shouldn't it be SocketAddr instead of String?

Reply:
No, see https://docs.rs/fortanix-sgx-abi/0.3.3/fortanix_sgx_abi/struct.Usercalls.html#networking